### PR TITLE
[Task-18][Backend] Search report by keyword & [Task-19][Frontend] Search report by keyword

### LINKED ISSUE #42: [Task-18][Backend] Search report by keyword
## Why

User can search using keyword

## Acceptance Criteria

Search working perfectly with tests

## Who Benefits?

Users

### LINKED ISSUE #43: [Task-19][Frontend] Search report by keyword
## Why

User can search using keyword

## Acceptance Criteria

Nice bootstrap search box

## Who Benefits?

Users

### PR DESCRIPTION
**Pull request by rafayet-monon**
Task Card - https://github.com/rafayet-monon/elixir_search_extractor/projects/1#card-64813100
                  - https://github.com/rafayet-monon/elixir_search_extractor/projects/1#card-64813147

## What happened 👀

User can search report by keyword

## Insight 📝

- Search form
- Search functionality

## Proof Of Work 📹

ElixirSearchExtractor (1)

### CODE REVIEW

File: lib/elixir_search_extractor/search_keyword/queries/keyword_query.ex [hunk 1]
```diff
@@ -10,4 +10,10 @@ defmodule ElixirSearchExtractor.SearchKeyword.Queries.KeywordQuery do
       where: file.user_id == ^user.id,
       order_by: fragment("? DESC", keyword.inserted_at)
   end
+
+  def filter_keywords(query, nil), do: query
```
Comment (hanam1ni) [inline lib/elixir_search_extractor/search_keyword/queries/keyword_query.ex:12]: Let's add test for this function

File: test/elixir_search_extractor/search_keyword/queries/keyword_query_test.ex [hunk 2]
```diff
@@ -0,0 +1,15 @@
+defmodule ElixirSearchExtractor.SearchKeyword.Queries.KeywordQueryTest do
+  use ExUnit.Case, async: true
+
+  alias ElixirSearchExtractor.SearchKeyword.Queries.KeywordQuery
+
+  describe "filter_keywords/2" do
+    test "it returns unmodified query if the second argument is nil" do
```
Comment (rosle) [inline test/elixir_search_extractor/search_keyword/queries/keyword_query_test.ex]:
```suggestion
    test "returns unmodified query if the second argument is nil" do
```

File: lib/elixir_search_extractor/search_keyword/queries/keyword_query.ex (hunk 1 above)
Comment (rafayet-monon) [reply lib/elixir_search_extractor/search_keyword/queries/keyword_query.ex:12]: @hanam1ni added in [6028f82](https://github.com/rafayet-monon/elixir_search_extractor/pull/44/commits/6028f8286c4f53f05257447453568337ba88a59a)

File: test/elixir_search_extractor/search_keyword/queries/keyword_query_test.ex (hunk 2 above)
Comment (rafayet-monon) [reply test/elixir_search_extractor/search_keyword/queries/keyword_query_test.ex]: @rosle removed in [5845596](https://github.com/rafayet-monon/elixir_search_extractor/pull/44/commits/5845596a9b46caf352b7ff1abd2c886664528970)

**Review by hanam1ni — APPROVED** [review]

**Review by rosle — APPROVED** [review]

**Pull request merged**